<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1450</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное окно</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>241</width>
      <height>871</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_left_menu">
     <item>
      <widget class="QFrame" name="frame_left_menu">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="pushButton_tasks">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>221</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Мои задачи</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_find_projects">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>209</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_profile">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>221</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Мой профиль</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_find_projects">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Поиск по проектам:</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget_projects">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>211</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_projects">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Проекты:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_create_project">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>221</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Создать проект</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_delete_project">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>780</y>
          <width>221</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Завершить проект</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_create_report">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>820</y>
          <width>221</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Экспорт отчета</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>1001</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_find_sort">
     <item>
      <widget class="QFrame" name="frame_right_up_menu">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_project_name">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>311</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Название проекта</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_find_by_tasks">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>40</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_project_find_tasks">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Поиск по задачам:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_project_description">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>10</y>
          <width>621</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>119</y>
      <width>285</width>
      <height>761</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_open" stretch="0">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_open_tasks" native="true">
       <property name="minimumSize">
        <size>
         <width>275</width>
         <height>600</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>275</width>
         <height>800</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_title_open_tasks">
          <property name="text">
           <string>Открыто</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_add_open_task">
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_open_tasks">
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>600</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>260</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>258</width>
             <height>661</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>119</y>
      <width>285</width>
      <height>761</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_progress" stretch="0">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_progress_tasks" native="true">
       <property name="minimumSize">
        <size>
         <width>275</width>
         <height>600</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>275</width>
         <height>800</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_title_progress_tasks">
          <property name="text">
           <string>В работе</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_progress_tasks">
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_progress_tasks">
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>600</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>260</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>258</width>
             <height>661</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>119</y>
      <width>285</width>
      <height>761</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_finish" stretch="0">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_finish_tasks" native="true">
       <property name="minimumSize">
        <size>
         <width>275</width>
         <height>600</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>275</width>
         <height>800</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_title_finish_tasks">
          <property name="text">
           <string>Готово</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_finish_tasks">
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_finish_tasks">
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>600</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>260</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>258</width>
             <height>661</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>119</y>
      <width>285</width>
      <height>761</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_close" stretch="0">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_close_tasks" native="true">
       <property name="minimumSize">
        <size>
         <width>275</width>
         <height>600</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>275</width>
         <height>800</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_title_close_tasks">
          <property name="text">
           <string>Закрыто</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_close_tasks">
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_close_tasks">
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>600</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>260</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>258</width>
             <height>661</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
